src/styles/card.css

```diff
@@ -0,0 +1,86 @@
+.card-block {
+  display: flex;
+  flex-wrap: wrap;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-block__image {
+  width: 160px;
+  margin: 32px auto 0;
+}
+
+.card-block__product-title {
+  margin: 40px 16px 0;
+  text-align: center;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
```

Comment:
You have 4 different colors in this task. Would be better if you will make an external file in which you will add variables for this colors and use them in you card.css.